resources/sdk/clisdkclient/extensions/transform_data/transform_data.go

```diff
@@ -42,8 +48,26 @@ func convertArrayOfObjectsToMap(data string) []map[string]interface{} {
 
 func ProcessTemplateFile(mp interface{}) string {
 	path := []string{TemplateFile}
-	name := filepath.Base(path[0])
-	tmpl := handleParse(template.New(name).Funcs(sprig.TxtFuncMap()).ParseFiles(path...))
+	fileName = filepath.Base(path[0])
+
+	if strings.Contains(TemplateFile, "github") {
+		rawURL := getFullUrlToRawFile(TemplateFile)
+
+		resp, err := http.Get(rawURL)
+		if err != nil {
+			logger.Fatal(err)
```

Comment:
It might be more helpful to preprend this and the following error with a message. Something like "Error downloading template file:" for this one would be helpful